test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Execution/LogMessageListenerTests.cs

```diff
@@ -92,21 +97,4 @@ public void DisposeShouldDisposeTraceListener()
     }
 
     #endregion
-
-    [TestMethod]
-    public void LogMessageListenerShouldCaptureLogMessagesInAllListeningScopes()
```

Comment:
This test was testing the behavior of `ns10ThreadSafeStringWriter` which was used only by PlatformServices.Portable. With real implementation the behavior is totally different and the test makes no sense.